src/openrct2/rct12/RCT12.cpp

```diff
@@ -858,3 +858,37 @@ ResearchItem RCT12ResearchItem::ToResearchItem() const
 
     return newResearchItem;
 }
+
+void HideQueuePaths()
```

Comment:
This doesn't need to iterate over all tile elements, it can be fixed during tile element import.